src/plugins/condition/components/Criterion.vue

```diff
@@ -118,21 +120,17 @@ export default {
             inputTypes: INPUT_TYPES
         }
     },
-    watch: {
-        telemetry: {
-            handler(newTelemetry, oldTelemetry) {
-                this.checkTelemetry();
-            },
-            deep: true
-        }
-    },
     computed: {
         setRowLabel: function () {
             let operator = this.trigger === 'all' ? 'and ': 'or ';
             return (this.index !== 0 ? operator : '') + 'when';
         },
         filteredOps: function () {
-            return [...this.operations.filter(op => op.appliesTo.indexOf(this.operationFormat) !== -1)];
+            if (this.operationFormat === 'all') {
+                return this.operations;
```

Comment:
Struggling to understand things a little here. Don't we want to filter the operations in both cases? What if the fields are *all* numbers?

Reply:
I'm going to look through this code and revisit the logic for these dropdowns. I'll update soon.

Reply:
Done.